### ISSUE TITLE
User Defined Snippets not working

### ISSUE THREAD

**Issue by BrandonRice** [Microsoft/vscode#17781]
- VSCode Version: Code 1.8.1 (ee428b0eead68bf0fb99ab5fdc4439be227b6281, 2016-12-19T14:49:23.350Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions:

|Extension|Author|Version|
|---|---|---|
|html-snippets|abusaidm|0.0.18|
|vscode-color|anseki|0.2.2|
|asp|ilich8086|0.0.1|
|htmltagwrap|bradgashler|0.0.3|
|vscode-html-css|ecmel|0.0.30|
|vscode-npm-script|eg2|0.1.4|
|vs-code-xml-format|fabianlauer|0.1.4|
|beautify|HookyQR|0.6.2|
|vscode-todo|MattiasPernhult|0.8.0|
|vscode-icons|robertohuertasm|4.2.0|
|Spell|seanmcbreen|0.9.1|
|asp|tron|0.0.1|
|JavaScriptSnippets|xabikos|1.2.0|
|html-css-class-completion|Zignd|1.0.3|

---

Please see below my user defined snippets for HMTL language:

```
{
    /*
         // Place your snippets for HTML here. Each snippet is defined under a snippet name and has a prefix, body and 
    	 // description. The prefix is what is used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
    	 // $1, $2 for tab stops, ${id} and ${id:label} and ${1:label} for variables. Variables with the same id are connected.
    	 // Example:
    	 "Print to console": {
    		"prefix": "log",
    		"body": [
    			"console.log('$1');",
    			"$2"
    		],
    		"description": "Log output to console"
    	}
    */
    "Tech Option": {
        "prefix": "techoption",
        "body": [
            "<%=strTechOut%>"
        ],
        "description": "Tech Option"
    },
    "Tech All": {
        "prefix": "techall",
        "body": [
            "<option value=\"%\">All</option>",
            "<%=strTechOut%>"
        ],
        "description": "Tech All"
    },
    "Center": {
        "prefix": "centerthis",
        "body": [
            "<Center>",
            "   ${Data}",
            "</Center>"
        ],
        "description": "Places a center tag with caller to enter data in code"
    },
    "myOption": {
        "prefix": "myoption",
        "body": [
            "<option value=${Value}>${Text}</option>"
        ],
        "description": "Places an option tag with callers to enter data in code"
    },
    "mySelect": {
        "prefix": "mySelect",
        "body": [
            "<select id=${id} name=${name} size=\"1\">",
            "   <option value=${Value}>${Text}</option>",
            "</select>"
        ],
        "description": "Places a select tag with option block including callers in code"
    },
    "myNoWrap": {
        "prefix": "myNoWrap",
        "body": [
            "&nbsp;"
        ],
        "description": "Places a no wrap tag in code"
    },
    "Validation": {
        "prefix": "Validation",
        "body": [
            "validation=${Validation}"
        ],
        "description": "Enters a validation script into code."
    },
    "MyInput": {
        "prefix": "MyInput",
        "body": [
            "<input type=${Type} name=${Name} id=${Name} value=${Value}>"
        ],
        "description": "Enters a full input tag into code."
    },
    "MyDocument":{
        "prefix": "MyDocument",
        "body": [
            "document.getElementById(${ID})"
        ],
        "description": "Enters document java tag into code"
    }
}
```

Up until the most recent update, if I typed "MyIn", intellisense would show me MyInput and allow me to TAB or ENTER to enter the snippet onto the page.

Now, if I type any of my User Defined Snippets, nothing comes up. See attached gif
![2016-12-23_10-51-25](https://cloud.githubusercontent.com/assets/6462859/21458682/2eb49c9e-c8fe-11e6-8193-257cd61b5687.gif)
.

**Comment by jrieken** [comment]
<img width="1209" alt="screen shot 2016-12-27 at 10 07 53" src="https://cloud.githubusercontent.com/assets/1794099/21496126/6a5f5330-cc1c-11e6-98f7-a0845915e441.png">

I cannot reproduce with the snippets and steps. Please clarify and update this issue? Any chance you have configured the `editor.snippetSuggestions`?

**Comment by BrandonRice** [comment]
I found the issue this morning.  I guess I didn't pay attention to the type
of language that VSCode was recognizing the page.  I had placed my snippets
in the HTML language but the page was being recognized as ASP.  I have
copied my code to the ASP language as well and the snippets work fine.
Thank you for looking into this and I apologize for the false alarm.

Thank you,

Brandon Rice
Web/Db Application Developer
Office: (817) 200-2374
